docker/test/fuzzer/run-fuzzer.sh

```diff
@@ -399,3 +398,10 @@ EOF
 esac
 
 exit $task_exit_code
+
+# Core dumps (see gcore)
+# Default filename is 'core.PROCESS_ID'
+for core in core.*; do
+    pigz $core
+    mv $core.gz /output/
+done
```

Comment:
By the way, I'm not sure that artifacts via `/output` works for `fuzzer`, is this correct?

Reply:
It's not: https://github.com/ClickHouse/ClickHouse/blob/73a593b67625aa9b478594f2f5d675e215347e25/tests/ci/ast_fuzzer_check.py#L95-L108

Reply:
Added uploading `core.gz`, and here is a worked example - https://s3.amazonaws.com/clickhouse-test-reports/33057/c43709a12fcca4452331464d31ace4df804e8796/fuzzer_astfuzzerdebug,actions//report.html